skrf/media/cpw.py

```diff
@@ -25,50 +24,198 @@
 
 class CPW(Media):
     """
-    A Coplanar Waveguide.
-
-    This class was made from the technical documentation [#]_ provided
-    by the qucs project [#]_ .
-    The variables  and properties of this class are coincident with
-    their derivations.
+    A coplanar waveguide transmission line is defined in terms of width,
```

Comment:
for the API generated by sphinx, I suggest the first line to be short:
```suggestion
    Coplaner waveguide.

    A coplanar waveguide transmission line is defined in terms of width,
```